src/scancode/api.py

```diff
@@ -252,6 +253,7 @@ def _licenses_data_from_match(
         result['homepage_url'] = lic.homepage_url
         result['text_url'] = lic.text_urls[0] if lic.text_urls else ''
         result['reference_url'] = license_url_template.format(lic.key)
```

Comment:
All these URLs should be empty for additional licenses for now. Later we could provide a way to include them in the data files.

Reply:
@KevinJi22 you wrote:

> also, for this, I don't think there's a way to identify if a license matches an external license vs. a license already in the licenseDB, right? So I'm not sure how we can set these fields to empty for additional licenses

Good point... may be we need to add some flag or data source field of sorts to the License and Rule? something that would indicate the provenance?
Some food for thoughts: a minimal approach could be an "is_builtin" field added to the License and Rule and that would be set automatically to False for additions and True for builtins?

Reply:
This would work as an approach but then I think we'd have to separate out loading the built in licenses/rules and the additional ones, which is a little less clean but I think that's the only way to do it. 

Basically, going back on this idea and treating licenses_data_dir as special:
> I wonder if it could be simpler to always consider multiple directories and not treat "licenses_data_dir" as special?